lib/globalize.rb

```diff
@@ -54,7 +54,7 @@ def default_fallbacks(for_locale = self.locale)
     end
     
     def fallbacks_for_empty_translations=(value = false)
-      return if [true, false].include? value
+      return if !!value != value # Only allow booleans
```

Comment:
Curious.. how does this improve on `[true, false].include?(value)`? That version only allows booleans too. :smile:

Reply:
Or do you mean this restricts what is passed to the method as `value`?

Reply:
if that's our intention, perhaps we should just guard:
```ruby
raise ArgumentError, 'fallbacks_for_empty_translations= can only be true or false' if [true, false].exclude?(value)
```